### ISSUE TITLE
Improve Custom Game settings and English translations

### ISSUE THREAD

**Issue by KonSola5** [26F-Studio/Techmino#1003]
Currently, the custom game menu looks like this:
![custom_game](https://github.com/26F-Studio/Techmino/assets/125081901/35c24ba8-a711-4098-8e42-45c706d69af6)

My suggestions:
## Fine tuning
- [ ] Change the various numerical adjustments to sliders, like the Next and Hold preview ones (might be hard with current slider implementation, especially if there are a lot of selections.

If sliders would be too difficult to implement:
- [ ] Add 19 and 21 as valid options for Field Height.
- [ ] Make Lock Delay adjustable with 1 frame steps between 0 and 16 frames.
- [X] Add 9 as a valid option for various other delay options.
- [X] Add 0 as a valid option for *Garbage* Buffer Limit (effectively disabling garbage in bot battles.)

## More fine tuning
- [ ] Enforced handling for a Custom Game - currently you need to change your global handling settings to modify handling for a Custom Game, despite game modes being able to enforce handling.
- [ ] Enforced "no smooth falling" and "no rotation center" for Classic rulesets.

## Other
- [ ] Add an ability to import/export current Custom Game settings to clipboard (currently you can only copy missions, boards and sequences)
- [ ] Change the "Normal Lock Reset" checkbox to a selection box, similar to the "Hold Mode" one:
> Lock Reset Mode: Move/Step

## English translations
- [x] Entry Delay → Entry Delay (ARE) *(for inexperienced players to explain that ARE is a Japanese term for Entry Delay)*
- [x] Buffer Limit → Garbage Buffer Limit *(could be confused with the Buffer Zone, the area above the skyline)*
- [x] Push Speed → Garbage Push Speed
- [X] O-spin → TRS O-spin *(or something else to signify that this affects the ability for 0 to transform in TRS)*
- [X] Rule Set → Ruleset

**Comment by MrZ626** [comment]
(partial resolved)

**Comment by ImpleLee** [comment]
Maybe this issue can be split into many smaller issues so related discussion/commit/etc can happen separately?

**Comment by shoucandanghehe** [comment]
> Maybe this issue can be split into many smaller issues so related discussion/commit/etc can happen separately?

Yes, just click on convert to issue to the right of the options.
Also some of these issues are simple, so I've labeled them as good first issue🥳
